cli/cmd/plugin/unmanaged-cluster/log/log.go

```diff
@@ -106,10 +109,13 @@ type Logger interface {
 
 // NewLogger returns an instance of Logger, implemented via CMDLogger.
 func NewLogger(tty bool, level int) Logger {
+	fd := int(os.Stdout.Fd())
+
 	return &CMDLogger{
 		tty:    tty,
 		level:  level,
 		output: os.Stdout,
+		termFd: fd,
```

Comment:
Seems like we could get this as needed in the progressf call, but things are usually such a short lived process that I suppose it's more efficient to open it once and pass it around rather than getting it on every progress update.

Reply:
True - I wanted to prevent a `syscall` every time we log progress (roughly once / second), so instead, storing it as part of the command logger struct made more sense. 

I also thought it might be something other APIs in the logging package want, but that's debatable.

Further, the likelihood of the terminal detaching and having it's file-descriptor change is unlikely.